polling_stations/templates/fragments/feedback_js.html

```diff
@@ -15,10 +15,23 @@
     });
 
     if ($('#feedback_form input[name=found_useful]:checked').val() === undefined) {
+
       $("#feedback_form .comments_container").hide();
+      $('<input type="radio" name="found_useful" value="PROBLEM" required data-toggle="button" id="report_problem" />'+
+        '<label class="link-button" for="report_problem">Report a problem with this page</label>'
```

Comment:
Also open to feedback on the text here (ping @JoeMitchell ). I think 'Report a problem with this page' is good but we could say 'Report a problem'  or 'There is something wrong with this page' (maybe that fits better as an 'answer' to 'Did you find this useful?' The context is:

![screenshot at 2018-03-23 14 18 42](https://user-images.githubusercontent.com/6025893/37836098-97c9855c-2ea9-11e8-8a60-d7319257474c.png)

Reply:
Quite tricky this bit, might just need to be out in the world and see what happens.

My guess is that 'There is something wrong with this page' better suits a factual error, whereas 'Report a problem' better suits a technical bug... But we're looking for both, I imagine?

Maybe 'Report a problem' because it's shorter? :)